### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by Megha-Joshi**
Please review my homepage which contains filter, navbar, footer, context of my ecom website.

### CODE REVIEW

File: src/cave-components/Footer/footer.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
+.footer-section {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 1.5rem;
+    background-color: #21256a;
```
Comment (zoyatanweer) [inline src/cave-components/Footer/footer.css]: use variable for color

File: src/cave-components/Homepage/homePage.css [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+.card-text {
+    color: var(--accent-color);
+    margin: 10px;
+}
+
+.res-image {
+    height: 28.125rem;
+}
+
+
```
Comment (zoyatanweer) [inline src/cave-components/Homepage/homePage.css:75]: remove spacing

File: src/cave-components/Homepage/homePage.css [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+.left-nav {
+    text-decoration: none;
+    color: var(--accent-color);
+    cursor: pointer;
+
```
Comment (zoyatanweer) [inline src/cave-components/Homepage/homePage.css:80]: remove spacing

File: src/public-css/navbar.css [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    align-items: center;
+}
+
+.btn-login {
+    margin: 10px;
+    min-width: 6.25rem;
+    cursor: pointer;
+    border-radius: 1.8rem;
+    padding: 0.8rem 1rem;
+
```
Comment (zoyatanweer) [inline src/public-css/navbar.css]: remove spacing

File: src/public-css/navbar.css [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+}
+
+.nav-header {
+    display: flex;
+    width: 100%;
+    height: 6.25rem;
+    justify-content: center;
+    align-items: center;
+    gap: 10rem;
+    padding: var(--container-padding);
```
Comment (zoyatanweer) [inline src/public-css/navbar.css:45]: appreciation : nice usage of var for padding here.

File: src/public-css/root.css [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+.outline-btn-color {
+    color: var(--accent-color);
+}
+
+.btn-info-outline:hover {
+    background-color: var(--bg-color);
+    text-decoration: underline;
+}
+
+
```
Comment (zoyatanweer) [inline src/public-css/root.css]: remove unused spacing

**Review by zoyatanweer — APPROVED** [review]
- code is clean and readable
- good way of writing comments
- good to merge

File: src/App.css [hunk 7]
```diff
@@ -1,6 +1,6 @@
-.App {
+/* .App {
   text-align: center;
-}
+} */
```
Comment (Jia-A) [inline src/App.css]: Remove the commented code.

File: src/Context/cartContext.js [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+                    cartState.cart.map((item) => item.id === action.payload.id ? 
+                    {...item, quantity: item.quantity + 1} : item)
+                    }
+        case "Decrease_quantity":
+            return {...cartState,
+                cart: 
+                    cartState.cart.map((item) => item.id === action.payload.id ? 
+                    {...item, quantity: item.quantity > 1 ? item.quantity - 1: item.quantity} : item)
+                    }
+                    case "Remove_from_cart":
```
Comment (Jia-A) [inline src/Context/cartContext.js]: Good practice is using capital letters for cases like "REMOVE_FROM_CART".

File: src/cave-components/Filter/filter.jsx [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+                Price : Low to High
+            </label>
+            <label for="h-to-l" class="disp-new-line sm-text">
+                <input type="radio" id="h-to-l" name="sorting" value="PRICE_HIGH_TO_LOW"
+                    checked={filterState.sortByFilter==="PRICE_HIGH_TO_LOW" } onClick={()=> filterDispatch({type:
+                "SORT_BY_PRICE", payload: "PRICE_HIGH_TO_LOW"})}/>
+                Price : High to Low
+            </label>
+        </div>
+    </aside>
```
Comment (Jia-A) [inline src/cave-components/Filter/filter.jsx]: Good use of semantic HTML.

File: src/public-css/root.css [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+.justify-align {
+    justify-content: center;
+    align-items: center;
+}
+
+.title {
+    margin: 0.625rem 0rem 1.25rem 0.625rem;
+}
+
+/* Override Component Library */
```
Comment (Jia-A) [inline src/public-css/root.css]: Nice use of comments

**Review by Jia-A — APPROVED** [review]
- Good use of comments for better understanding of overwritten component library properties.
- Clean code is written.
- Good file and code structure is used.
- Suggestion : Try to keep the pull requests small by keeping images apart.

File: src/cave-components/Footer/footer.css (hunk 1 above)
Comment (Megha-Joshi) [reply src/cave-components/Footer/footer.css]: Okay

File: src/cave-components/Homepage/homePage.css (hunk 2 above)
Comment (Megha-Joshi) [reply src/cave-components/Homepage/homePage.css:75]: Okay

File: src/App.css (hunk 7 above)
Comment (Megha-Joshi) [reply src/App.css]: Okay! will do it

File: src/Context/cartContext.js (hunk 8 above)
Comment (Megha-Joshi) [reply src/Context/cartContext.js]: Okay

**Pull request merged**